moxy/org.eclipse.persistence.moxy/src/test/java/org/eclipse/persistence/testing/jaxb/casesensitivity/JAXBCaseInsensitivityTestCase.java

```diff
@@ -39,8 +39,8 @@
 public class JAXBCaseInsensitivityTestCase extends junit.framework.TestCase {
 
     private static final URL CUSTOMER_FILE_URL = Thread.currentThread().getContextClassLoader().getResource("org/eclipse/persistence/testing/jaxb/casesensitivity/customer.xml");
-    private static final Class[] CAMEL_CASE_CUSTOMER = new Class[]{CustomerImpl.class};
-    private static final Class[] UPPER_CASE_CUSTOMER = new Class[]{org.eclipse.persistence.testing.jaxb.casesensitivity.otherCase.CustomerImpl.class};
+    private static final Class<?>[] CAMEL_CASE_CUSTOMER = new Class[]{CustomerImpl.class};
```

Comment:
this won't fix all warnings, one needs sth like for that:
```
@SuppressWarnings({"unchecked"})
private static final Class<?>[] CAMEL_CASE_CUSTOMER = (Class<?>[]) new Class[]{CustomerImpl.class};
```

Reply:
This changeset was removed from PR.